extension/src/decoding.rs

```diff
@@ -50,3 +52,14 @@ impl<
 	type Error = Error;
 	const LOG_TARGET: &'static str = Logger::LOG_TARGET;
 }
+
+/// Default processor implementation which just passes through the value unchanged.
+pub struct IdentityProcessor;
```

Comment:
Wonder why is this named `IndentityProcessor`?

Reply:
Not a great name, I admit. It was partially inspired by the [Identity hasher](https://docs.substrate.io/build/runtime-storage/#hashing-algorithms) in Substrate. Any better suggestions? Could just be `Identity` instead.

Reply:
Tbh, I think it's a great name! Quite clear what it does with inspiration from math, and postfixing with Processor makes quite easy to recognize what it's doing when reading tests etc.

Reply:
This was already moved in the tests PR. I did shorten it to `Identity` for simplicity, which implements the `Processor` trait.

Reply:
As mentioned, this is now available at https://github.com/r0gue-io/pop-node/blob/39f16e1459e75b86d07414d7e3c70e21dc97d8e0/extension/src/decoding.rs#L60 as desired. Resolving this, let me know if anything else required.